File: python/2/13.py
import pandas as pd

df1 = pd.read_csv('../../output/col1.txt', header=None, delimiter='\t')
df2 = pd.read_csv('../../output/col2.txt', header=None, delimiter='\t')
merged_df = pd.concat([df1, df2], axis=1)
merged_df.to_csv('../../output/hightemp13.txt', index=False, header=False, sep='\t')

File: ruby/2/13.rb
col1 = "col1.txt"
col2 = "col2.txt"

prefs = File.readlines(col1).map(&:chomp)
cities = File.readlines(col2).map(&:chomp)

output = [prefs, cities].transpose.map { |row| row.join("\t") }

File.open("hightemp13.txt", "w") do |file|
  file.puts output
end

# String#chomp -> String
# 末尾から改行コードを取り除く
